src/reactionsystem_conversions.jl

```diff
@@ -784,8 +778,8 @@ struct JumpInputs{S <: MT.JumpSystem, T <: SciMLBase.AbstractODEProblem}
 end
 
 """
-    jumpinput = JumpInputs(rs::ReactionSystem, u0map, tspan,
-                    pmap = DiffEqBase.NullParameters;
+    jumpinput = JumpInputs(rs::ReactionSystem, u0, tspan,
+                    p = DiffEqBase.NullParameters;
```

Comment:
use the names in the docstring we use in the function

Reply:
(happy to revert, but thought I saw an inconsistency when I ctrl+f searched for `u0map` and thought I might as well update it)